musicbot/bot.py

```diff
@@ -717,7 +717,13 @@ def _get_owner_voice_channel(self):
                     continue
 
                 if args:
-                    arg_value = args.pop(0)
+                    arg_value = None
+
+                    if len(args) > 1:
```

Comment:
Wouldn't this affect commands that have more than 1 argument? Which I believe would be the permissions ones or adding/removing from blacklist now.